frontend/public/components/operator-hub/operator-hub-page.tsx

```diff
@@ -34,7 +32,7 @@ export const OperatorHubList: React.SFC<OperatorHubListProps> = (props) => {
       subscription: subscriptionFor(subscription.data)(operatorGroup.data)(pkg.status.packageName)(namespace),
       // FIXME: Just use `installed`
       installState: installedFor(subscription.data)(operatorGroup.data)(pkg.status.packageName)(namespace) ? 'Installed' : 'Not Installed',
-      imgUrl: iconObj ? `data:${iconObj.mediatype};base64,${iconObj.base64data}` : operatorImg,
```

Comment:
The `CatalogTile` component from `patternfly-react` [doesn't support icon image fallback](https://github.com/patternfly/patternfly-react/issues/2739).